### PR TITLE
Refs #12812 - changes related to job_category

### PR DESCRIPTION
**Pull request by ares**
@adamruzicka anything you'd like to add to description format?

### CODE REVIEW

File: plugins/foreman_remote_execution/nightly/index.md [hunk 1]
```diff
@@ -228,24 +228,17 @@ stands for variables which you define in second tab called Job.
...
-
-This is more useful when more than one provider is available. For example, you
-might have both an SSH and an Ansible template that implements the "Install
-Package" job. It turned out that having two names is so complicated
-that we decided to merge template and job name in some upcoming version.
-
-A description format field is a template for job name. Imagine you run job
-template named "Install package" 10 times, each time with different package
+autocomplete field called **Job category**. This groups job template together
+and makes orientation in huge number of template easier.
```
Comment (adamruzicka) [inline plugins/foreman_remote_execution/nightly/index.md]: `templates`

File: plugins/foreman_remote_execution/nightly/index.md [hunk 2]
```diff
@@ -228,24 +228,17 @@ stands for variables which you define in second tab called Job.
...
+A description format field is a template for job description. Imagine you run
+a template named "Install package" 10 times, each time with different package
 name specified by user input (more about this below). On job overview page
 you'd see the same "Install package" job and you'd have to inspect provided
 inputs to find the one that installed let's say httpd package. Using
-description format you set your own rules for naming which can use input
-values as part of job names. In our example install package example a good
-description format would be following
+description format you set your own rules for creating the description.
+Input values can be used as part of it. In our example install package example,
```
Comment (adamruzicka) [inline plugins/foreman_remote_execution/nightly/index.md]: This sentence has too many examples, maybe "A good description format for our package install example would be..."?

File: plugins/foreman_remote_execution/nightly/index.md [hunk 3]
```diff
@@ -429,13 +419,13 @@ After you cancel the job, it cancels all future executions as well.
 ## 3.4 Jobs list
 
 You can find all Jobs when you navigate to Monitor > Jobs. A table
-lists all jobs from history to future. You can search jobs by **Job name** with
+lists all jobs from history to future. You can search jobs by description with
 a queries like
```
Comment (adamruzicka) [inline plugins/foreman_remote_execution/nightly/index.md]: `with a query like` or `with queries like`

**Comment by adamruzicka** [comment]
I added some inline comments, not exactly related to description format.
Also since we already have the future execution implemented could you change the first sentence of [this paragraph](https://github.com/ares/theforeman.org/commit/c5b4aa77b0e8a7f395a09f9020de6ee20db9885f#diff-2c2ec392cf6d40d526169d7aab53cde3R336) to something like this

> Dynamic query is a powerful tool when used with scheduled or recurring executions.

File: plugins/foreman_remote_execution/nightly/index.md (hunk 1 above)
Comment (stbenjam) [inline plugins/foreman_remote_execution/nightly/index.md]: Also `s/orientation in/managing a/` sounds better imho

**Comment by ares** [comment]
Fixed all including the dynamic search.

**Comment by stbenjam** [comment]
Looks ok to me, just my comment here https://github.com/theforeman/theforeman.org/pull/519#discussion_r50277255

**Comment by ares** [comment]
>  just my comment here #519 (comment)

fixed

**Comment by adamruzicka** [comment]
:+1:

**Pull request merged**